faqs/how-do-i-get-involved.md

```diff
@@ -0,0 +1,10 @@
+---
+title: How can I get involved
+weight: 9.1
+attribution:
+  name: sum guy on reddit
+  link: /u/twoofspades
+---
+
+See [Rayonism](https://rayonism.io/) on how to contribute. If your a dapp developer then consider deploying your dapp on on the merge testnet during the ethglobal hackathon (April 16 - May 14). If your a rollup project and would like to see a glimpse of the future? Then make use of sharded data on the merge testnet.
```

Comment:
- Please replace instances of "your" with "you're"
- Can you provide a link for the ethglobal hackathon? Also, please use proper casing for name (ETHGlobal)
- For your last two sentences, please combine into one.